viewer/src/locales/Swedish.js

```diff
@@ -35,15 +42,70 @@ export default {
         "Klistra in en URL till en .djvu-fil här",
     "Open URL":
         "Öppna URL",
+    'Enter a valid URL (it should start with "http(s)://")': // an alert shown when you try to open an empty URL
+        'TODO (sv): Enter a valid URL (it should start with "http(s)://")',
```

Comment:
But why haven't you translated it?

Reply:
I missed this line when looking for translated strings and did not sent to the translator. Sent it, now will wait till I'll get response. OK, either remove "TODO (sv): " or wait till my next commit.

Reply:
Closed pull request, will resend after I'll get the translated string.